src/NServiceBus.NHibernate/TimeoutPersisters/NHibernateTimeoutStorage.cs

```diff
@@ -54,10 +56,14 @@ protected override void Setup(FeatureConfigurationContext context)
                 }
             }
 
+            var timeoutsCleanupExecutionInterval = 
+                context.Settings.GetOrDefault<TimeSpan?>("NHibernate.Timeouts.CleanupExecutionInterval") ?? TimeSpan.FromMinutes(2);
```

Comment:
Yeah, like `unlikely event of losing cabin pressure` :smiley: Fine. Should we document that with NH persistence the worst case delay of timeout delivery when idle is 3 minutes? This is not a blocker for this pull. I am :+1: for beginning the hotfix procedure.